IOPool/Streamer/interface/FRDEventMessage.h

```diff
@@ -123,8 +125,9 @@ struct FRDEventHeader_V1 {
 
 const uint16 FRDEVENT_MASK_ISGENDATA = 1;
 
-const uint32 FRDHeaderVersionSize[6] = {
-    0, 2 * sizeof(uint32), (4 + 1024) * sizeof(uint32), 7 * sizeof(uint32), 8 * sizeof(uint32), 6 * sizeof(uint32)};
+const size_t FRDHeaderMaxVersion = 6;
+const std::array<uint32,FRDHeaderMaxVersion+1> FRDHeaderVersionSize = {
```

Comment:
How about
```suggestion
constexpr size_t FRDHeaderMaxVersion = 6;
constexpr std::array<uint32,FRDHeaderMaxVersion+1> FRDHeaderVersionSize = {
```
? (although the practical impact may be negiligble)

Reply:
seems fine, will add it with code checks fix.